lib/manual_input_sheet.rb

```diff
@@ -0,0 +1,45 @@
+class ManualInputSheet < GoogleSheet
+  SHEET_ID = "16R7F6Y35M4Pe9FkcL51pbIiV-4ZkdXef9uZtVADszWY"
```

Comment:
Let's move this into an environment variable.

Reply:
Sure!